packages/server/lambdas/alfresco/list-files.ts

```diff
@@ -0,0 +1,79 @@
+import { CategoryDataBase } from '@prisma/client';
+import { ALBEvent, Context } from 'aws-lambda';
+import axios from 'axios';
+
+import { getRataExtraLambdaError, RataExtraLambdaError } from '../../utils/errors';
+import { log } from '../../utils/logger';
+import { findEndpoint, getAlfrescoOptions, getAlfrescoUrlBase } from '../../utils/alfresco';
+import { getUser, validateReadUser } from '../../utils/userService';
+import { DatabaseClient } from '../database/client';
+import { searchQueryBuilder } from './searchQueryBuilder';
+import {
+  AdditionalFields,
+  IParentSearchParameter,
+  QueryLanguage,
+  SearchParameterName,
+} from './searchQueryBuilder/types';
+
+const searchByTermWithParent = async (uid: string, alfrescoParent: string, page: number, language: QueryLanguage) => {
+  try {
+    const parent: IParentSearchParameter = {
+      parameterName: SearchParameterName.PARENT,
+      parent: alfrescoParent,
+    };
+
+    const bodyRequest = searchQueryBuilder({
+      searchParameters: [parent],
+      page: page,
+      language: language,
+      additionalFields: [AdditionalFields.PROPERTIES],
+    });
+    const alfrescoAPIUrl = getAlfrescoUrlBase();
+    const options = await getAlfrescoOptions(uid, { 'Content-Type': 'application/json;charset=UTF-8' });
+
+    const response = await axios.post(`${alfrescoAPIUrl}/search`, bodyRequest, options);
+    return response.data;
+  } catch (err) {
+    throw err;
+  }
+};
+
+const database = await DatabaseClient.build();
+
+let fileEndpointsCache: Array<CategoryDataBase> = [];
+
```

Comment:
Should we document by commenting in each `handleRequest` function what endpoint, method, body request structure,...? It'll be easier for team members to test

Reply:
This has been done now. Please share your thoughts!